desc/objectives/_stability.py

```diff
@@ -603,6 +610,12 @@ def compute(self, params, constants=None):
 
         lambda0, w0, w1 = constants["lambda0"], constants["w0"], constants["w1"]
 
+        # If I used constants["Neigvals"], I get an error about static argnums
```

Comment:
this is poor naming on our part. `constants` means constant during optimization, not the same as "constant" aka "static" in jax calculations. using `self._Neigvals` is fine, I think in the future we might get rid of `constants` since it was sort of a hack to get around some jax pytree stuff that has since been resolved.

Reply:
yigit is handling the deprecation so marking as resolved